src/formattedcode/output_doc.py

```diff
@@ -0,0 +1,326 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+import docx
```

Comment:
Some tests seem to be failing because docx cannot be imported.

See : https://ci.appveyor.com/project/nexB/scancode-toolkit/builds/39540630#L2257

You have added `python-docx` to requirements.txt but you also need to add it at setup.cfg [here](https://github.com/nexB/scancode-toolkit/blob/develop/setup.cfg#L54)